### ISSUE TITLE
Bitcode error on iOS 9.0 (Xcode 7 Beta 6)

### ISSUE THREAD

**Issue by tomtaylor** [mapbox/mapbox-gl-native#2205]
When compiling my application for iOS 9.0 (Xcode 7 Beta 6), I see the following build error:

```
 ld: '/Users/tom/projects/CityCyclist/app/Pods/Mapbox-iOS-SDK/libMapbox.a(SMCalloutView.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture armv7
```

I'm using MapboxGL through Cocoapods, setting the podspec to `https://raw.githubusercontent.com/mapbox/mapbox-gl-native/ios-v2.1.0-pre.2/ios/Mapbox-iOS-SDK.podspec`.

Let me know if I can provide any more info.

**Comment by friedbunny** [comment]
This bitcode issue will be because we're not building our static library with Xcode 7. [Travis just announced beta support for Xcode 7](http://blog.travis-ci.com/2015-08-26-xcode-70-beta-is-now-available/) and iOS 9 should ship around the second week of September, so it would be nice to get this moving.

@tomtaylor In the meantime, set `ENABLE_BITCODE` to `NO` in your Xcode project settings.

**Comment by tomtaylor** [comment]
Thanks @friedbunny - I had to add the following to my `Podfile`, otherwise Xcode still tried to use Bitcode for the dependencies:

```ruby
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
    end
  end
end
```

**Comment by incanus** [comment]
Keep your eyes peeled for 2.1.0 shortly @tomtaylor, which should fix this when we land https://github.com/mapbox/mapbox-gl-native/pull/2238 and build the release with Xcode 7.

**Comment by friedbunny** [comment]
We've had to remove bitcode support and release v2.1.2 because of Xcode 7 issues: #2332

**Comment by EddyVerbruggen** [comment]
Thanks for the `Podfile` tip, @tomtaylor :+1:

**Comment by thiagoross** [comment]
The Podfile tip didn't work for me here using Xcode 7, I had problems with others libs. I'm still waiting the new Mapbox version to start to know this beauty

**Comment by friedbunny** [comment]
@thiagoross You may need to wipe out your `Pods/` directory and then do a fresh `pod install`, otherwise I'm not sure that CocoaPods will do the proper configuration.

**Comment by 1ec5** [comment]
Any build after #2903 should come with proper support for Bitcode in Xcode 7. If you’re open to using a prerelease, that’d be [v3.0.0-pre.4](https://github.com/mapbox/mapbox-gl-native/releases/tag/ios-v3.0.0-pre.4) or any more recent build.

**Comment by incanus** [comment]
Which, BTW, you could get via a `Podfile` line like so: 

```ruby
pod 'Mapbox-iOS-SDK', :podspec => 'https://raw.githubusercontent.com/mapbox/mapbox-gl-native/ios-v3.0.0-pre.4/ios/Mapbox-iOS-SDK.podspec'
```

**Comment by 1ec5** [comment]
Closing per https://github.com/mapbox/mapbox-gl-native/issues/2205#issuecomment-153870974. A Bitcode-less build will also be available via #3003.
